changelogs/fragments/add_changed_deps_to.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "Adds a 'changed_deps' option to the portage module, which allows users to rebuild packages that have changed ebuild dependencies, corresponding to 'emerge --changed-deps'."
```

Comment:
Please refer to the standard formatting for the changelog fragments' entries:

https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to-format

The outcome of what you write will be published in the collection changelog:

https://docs.ansible.com/ansible/latest/collections/community/general/changelog.html